App/screens/ProofRequest.tsx

```diff
@@ -194,37 +250,105 @@ const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route })
   const connection = connectionRecordFromId(proof.connectionId)
 
   return (
-    <View style={styles.container}>
-      <ModularView
-        title={proof.requestMessage?.indyProofRequest?.name || connection?.alias || connection?.invitation?.label}
-        content={
-          <FlatList
-            data={retrievedCredentialsDisplay}
-            keyExtractor={(credential) => credential.name}
-            renderItem={({ item: credential }) => (
-              <Label title={credential.name} subtitle={credential.value} label={credential.credentialDefinitionId} />
+    <FlatList
+      ListHeaderComponent={() => (
+        <View style={styles.headerContainer}>
+          {/* <View style={styles.headerLogoContainer}>
```

Comment:
Ax the dead code?

Reply:
Sorry, will remove that in an updated PR